M/mrfDepth/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mrfDepth"
+version = v"1.0.14"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cran/mrfDepth.git", "6954d87f8ea3c87a2d945676233bf849af5aacd1")
+]
+
+# Bash recipe for building across all platforms
+# License is GPL ( >= 2 ) according to DESCRIPTION
+script = raw"""
+cd ${WORKSPACE}/srcdir/mrfDepth/src
+gfortran -shared -fPIC *.f -o libmrfDepth.${dlext}
+cp libmrfDepth.${dlext} ${libdir}
+install_license ${WORKSPACE}/srcdir/mrfDepth/DESCRIPTION
```

Comment:
License parsing tools will choke on this file. Might be better to do something like `install_license /usr/share/licenses/GPL-2.0+` ? With a comment linking to the description file?